src/types/index.ts

```diff
@@ -387,14 +387,30 @@ export type DataPointTemperature = Temperature & {
   type: DataPointType.TEMPERATURE;
   timestamp: string;
   precision: Temperature;
+} & {
+  value?: Power;
 };
 
-export type DataPoint = DataPointPercentage | DataPointTemperature;
-
-export type ZoneActivityDataPoints = {
-  heatingPower: DataPoint;
+export type DataPointPower = {
```

Comment:
How do you feel about this being a `PowerDataPoint` to match the heating one?

Reply:
Will fix it